kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -2753,6 +2861,431 @@ def get_vm_fencer_rule(self, req):
+        for n in list(parent):
+            if n.tag.endswith('}%s' % node_name):
+                return n
+
+        return None
+
+    def _parse_enable_ha_from_xml(self, xml, vm_uuid):
+        if not xml:
+            return None
+
+        try:
+            root = etree.fromstring(xml)
```

Ruff (0.15.2)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)